components/list/list-item-placement-marker.js

```diff
@@ -0,0 +1,29 @@
+import { html, LitElement } from 'lit-element/lit-element.js';
+
+class ListItemPlacementMarker extends LitElement {
+	static get properties() {
+		return {
+			color: { type: String },
+			displayed: {type: Boolean}
+		};
+	}
+
+	constructor() {
+		super();
+		this.color = "blue";
```

Comment:
what default color should be ?

Reply:
http://design.d2l/style/colour/
Celestine
You can do this the way Maya mentioned with `var(--d2l-color-celestine)` in css